src/app/test_executive/zkapps_timing.ml

```diff
@@ -258,8 +262,12 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
       section "Waiting for snapp with transfer from timed account that succeeds"
         (wait_for_snapp parties_transfer_from_timed_account)
     in
-    let%bind after_balance =
-      get_account_balance ~logger node timing_account_id
+    (* let%bind after_balance =
```

Comment:
rm commented code here and everywhere else introduced in this PR?

Reply:
yes of course, will do